index.d.ts

```diff
@@ -123,6 +123,35 @@ declare namespace boxen {
 		@default 'left'
 		*/
 		readonly align?: 'left' | 'right' | 'center';
+
+		/**
+		Display a title on the top of the box.
+		If needed the box will enlarge to fit the title.
+
+		@example
+		```
+		console.log(boxen('foo bar foo bar', {title: 'title'}));
+		// ┌title──────────┐
+		// │foo bar foo bar│
+		// └───────────────┘
+		```
+		*/
+		readonly title?: string;
+
+		/**
+		The alignment of the box title.
+
+		@default 'left'
+
+		@example
+		```
+		console.log(boxen('foo bar foo bar', {title: 'title', alignTitle: 'right'}));
+		// ┌────────title┐
+		// │foo bar foo  │
+		// └─────────────┘
+		```
+		*/
+		readonly titleAlign?: 'left' | 'right' | 'center';
```

Comment:
```suggestion
		readonly titleAlignment?: 'left' | 'right' | 'center';
```

I also want to do something about `align`. It should have been `alignment` and it might also be worth making it more specific now. Something like `textAlignment`.

Reply:
I see, although I am concerned that changing it now could break people's code. Maybe should we keep the existing 'align' and mark is as deprecated ?

Reply:
> Maybe should we keep the existing 'align' and mark is as deprecated ?

Yeah, we could keep it in code and TS for now, and just remove it from the docs.

Reply:
Done.